client/src/components/DataFiles/DataFilesModals/DataFilesUploadModal.js

```diff
@@ -102,7 +110,7 @@ const DataFilesUploadModal = ({ className, layout }) => {
     >
       <ModalHeader toggle={toggle}>Upload Files</ModalHeader>
       <ModalBody styleName={containerStyleNames}>
-        <div styleName="dropzone">
+        <div styleName="dropzone" disabled={disabled}>
```

Comment:
instead of creating and using a new React state with `useState`, I think we can figure out if the modal is uploading from the redux state, via `state.files.operationStatus.upload` and watching for any `status` there that is `UPLOADING`.

Also, let's disable when there are no files selected.

Reply:
Right, instead using `useSelector` to connect to a portion of the redux state, like [here](https://github.com/TACC/Core-Portal/blob/main/client/src/components/_common/SystemsPushKeysModal/SystemsPushKeysModal.js#L135)